src/components/InForm/InForm.tsx

```diff
@@ -11,8 +11,8 @@ import {
   Modal,
 } from "@fluentui/react";
 import { makeStyles } from "@fluentui/react-components";
-import React, { ChangeEvent, FormEvent, useState } from "react";
-import { PeoplePicker } from "../PeoplePicker/PeoplePicker";
+import React, { ChangeEvent, FormEvent, useContext, useState } from "react";
```

Comment:
Should probably get rid of the React import, and only import the components we need.
Looks like you only need to add FunctionComponent and useEffect.

Reply:
Good suggestion.  Updated all files that did React import to utilize selective imports